com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -24,7 +27,10 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem
 #pragma warning disable IDE1006 // disable naming rule violation check
 
         // RuntimeAccessModifiersILPP will make this `public`
-        internal static readonly Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>> __rpc_func_table = new Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>>();
+        internal delegate void RpcReceive(NetworkBehaviour behaviour, ref FastBufferReader reader, __RpcParams parameters);
```

Comment:
should probably be named as `RpcReceiveEventHandler`.

see: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#names-of-common-types

> ✔️ DO add the suffix "EventHandler" to names of delegates that are used in events.
> ✔️ DO add the suffix "Callback" to names of delegates other than those used as event handlers.
> ❌ DO NOT add the suffix "Delegate" to a delegate.

Reply:
Done.